src/mlstacks/terraform/aws-remote-state/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "region" {
+  description = "The region to deploy resources to"
+  default     = "eu-north-1"
```

Comment:
You're probably right, though I think in the past lots of things were set as defaults so as to prevent people from having to specify too much. But in the new setup I think we can get away with it.